packages/crw-client/Cargo.toml

```diff
@@ -12,16 +12,21 @@ keywords = ["blockchain", "cosmos", "cosmos-rust-wallet"]
 crate-type = ["cdylib", "lib"]
 
 [dependencies]
-cosmos-sdk-proto = { version = "0.3.0"}
+cosmos-sdk-proto = { version = "0.5.0"}
 prost = { version = "0.7.0"}
 prost-types = { version = "0.7" }
-reqwest = { version = "0.11.0", features = ["blocking", "json"]}
+reqwest = { version = "=0.10.10", features = ["json"] }
```

Comment:
Why exactly the 0.10.10 instead of the latest 0.11.4? Can we use the latest one?

Reply:
0.11.0 onwards was not wasm compatible, cannot remember the exact issue - I can find out if you need to know